locales/en-US.json

```diff
@@ -266,5 +266,25 @@
     "search": "Search",
     "cart": "Cart",
     "account": "Account"
+  },
```

Comment:
is-it translated for the other translations files ?

Reply:
translations refactored, account.address translations used